README.md

```diff
@@ -130,26 +140,28 @@ npm run start
-| CID_SUBDOMAINS          | false                       | Enables Bzz.link subdomain translation functionality for CIDs.                                             |
-| ENS_SUBDOMAINS          | false                       | Enables Bzz.link subdomain translation functionality for ENS.                                              |
-| REMOVE_PIN_HEADER       | true                        | Removes swarm-pin header on all proxy requests.                                                            |
-| `LOG_LEVEL`             | info                        | Log level that is outputted (values: `critical`, `error`, `warn`, `info`, `verbose`, `debug`)              |
-| POSTAGE_EXTENDSTTL      | false                       | Enables extends TTL feature. Works along with POSTAGE_AMOUNT                                               |
-| EXPOSE_HASHED_IDENTITY  | false                       | Exposes `x-bee-node` header, which is the hashed identity of the Bee node for identification purposes      |
+| Name                     | Default Value               | Description                                                                                                |
+| -----------------------  | --------------------------- | ---------------------------------------------------------------------------------------------------------- |
+| BEE_API_URL              | http://localhost:1633       | URL of the Bee node API                                                                                    |
+| BEE_DEBUG_API_URL        | http://localhost:1635       | URL of the Bee node Debug API. Required for postage stamps autobuy and hashed identity header.             |
+| AUTH_SECRET              | undefined                   | Authentication secret, disabled if not set (this secret is checked in the request header `authorization`). |
+| HOSTNAME                 | localhost                   | Hostname of the proxy. Required for Bzz.link support.                                                      |
+| PORT                     | 3000                        | Port of the proxy.                                                                                         |
+| POSTAGE_STAMP            | undefined                   | Postage stamp that should be used for all upload requests. If provided, the autobuy feature is disabled.   |
+| POSTAGE_DEPTH            | undefined                   | Postage stamp depth to be used when buying new stamps or selecting existing stamps.                        |
+| POSTAGE_AMOUNT           | undefined                   | Postage stamp amount to be used when buying new stamps or selecting existing stamps.                       |
+| POSTAGE_USAGE_THRESHOLD  | 0.7                         | Usage percentage at which new postage stamp will be bought (value between 0 and 1).                        |
```

Comment:
Also document its role in dilute mode please

Reply:
I don't use `POSTAGE_USAGE_THRESHOLD` to dilute the stamp. I use the stamp amount and depth to extend its capacity. Can you elaborate more on this, please?

Reply:
```
filterUsableStampsExtendsCapacity(stamps: PostageBatch[], usageThreshold: number): PostageBatch[]
```

This function uses `usageThreshold`, which is made from `Number(POSTAGE_USAGE_THRESHOLD || DEFAULT_POSTAGE_USAGE_THRESHOLD)`

Reply:
Oh I see, yes the `usageThreshold` variable is used to get the list of stamps that have a _usage_ more extensive than that threshold value and it need to be documented on the README according to the feature that it's been use